waveform_editor/gui/io/manager.py

```diff
@@ -57,7 +57,7 @@ def create_new_file(self):
         self.open_file = None
 
     def _confirm_unsaved_changes(self, action, message):
-        if self.main_gui.config.has_changed or self.main_gui.editor.has_changed():
+        if self.main_gui.config.has_changed or self.main_gui.editor.has_changed:
```

Comment:
`editor.has_changed` is a reactive expression, which will always evaluate True-ish

Let's make it a param again so this code keeps working :)